tests/test_mosaic.py

```diff
@@ -0,0 +1,118 @@
+import os
+import pytest
+import random
+import numpy as np
+import xarray as xr
+from click.testing import CliRunner
+from FMSgridtools.shared.mosaicobj import MosaicObj 
+from FMSgridtools.make_mosaic.make_mosaic import make_mosaic
+
+DEFAULT_GRID_SIZE = 48
+DEFAULT_TILE_NUMBER = 1
+
+gridfiles = [f'grid.tile{x}.nc' for x in range(6)]
+gridtiles = [f'tile{x}' for x in range(6)]
+tile_number = DEFAULT_TILE_NUMBER
+output = 'test_mosaic.nc'
+tilefile = 'tile1.nc'
+
+
+def test_create_tile():
+    xstart, xend = 0, 360
+    ystart, yend = -90, 90
+
+    x = np.arange(xstart, xend, dtype=float)
```

Comment:
would it be better to do dtype=np.float64 here instead of astype-ing on DataArray?